remoting/remoting-triple/src/main/java/com/alipay/sofa/rpc/tracer/sofatracer/TripleTracerAdapter.java

```diff
@@ -81,6 +82,10 @@ public static void beforeSend(SofaRequest sofaRequest, ConsumerConfig consumerCo
         header.put(RemotingConstants.HEAD_METHOD_NAME, sofaRequest.getMethodName());
         header.put(RemotingConstants.HEAD_TARGET_SERVICE, sofaRequest.getTargetServiceUniqueName());
         header.put(RemotingConstants.HEAD_TARGET_APP, sofaRequest.getTargetAppName());
+        Map<String, Object> requestProps = sofaRequest.getRequestProps();
```

Comment:
Consider the priority of each header key. For example,  `RemotingConstants.HEAD_TARGET_SERVICE` uses the value of `sofaRequest.getTargetServiceUniqueName()`, can it be replaced by `sofaRequest.getRequestProps()`?

Reply:
There is indeed the problem of coverage. But I think this is reasonable. Users can customize the key to overwrite the original value here, and the original logic of bolt is the same. Of course, it is generally not recommended to use this method to overwrite.